templates/app/env

```diff
@@ -7,3 +7,4 @@ FOREST_ENV_SECRET=<%= forestEnvSecret %>
 <% if (devRenderingId) {%>DEV_RENDERING_ID=<%= devRenderingId %><% } %>
 <% if (dbSchema) {%>DATABASE_SCHEMA=<%= dbSchema %><% } %>
 <% if (port) {%>PORT=<%= port %><% } %>
+<% if (frontendHost) {%>FRONTEND_HOST=<%= frontendHost %><% } %>
```

Comment:
Same here. I prefer to set the environment variable in the dotenv manually or run my lumber back office application with: `FRONTEND_HOST=http://localhost:4200 npm start` if you don't want to set it into the .env